### PR TITLE
avs kql updates

### PR DESCRIPTION
**Pull request by jchancellor-ms**
# Overview/Summary

This pull requests adds resource graph queries for AVS-3, AVS-4, AVS-5, and AVS-6.  It also makes minor changes to the recommendation text. (Cleans up some grammar and removes random double spaces).  Finally, it removes the avs-1.kql file that erroneously got moved into the root of the AVS folder.

## Related Issues/Work Items

Replace this with a list of related GitHub Issues and/or ADO Work Items (Internal Only)

- To associate a GitHub Issue, use a [key word](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) preceded with the GitHub issue number.
- To associate an ADO Work Item, use the key word `AB#` succeeded with the [ADO Work Item ID](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword).

## This PR fixes/adds/changes/removes

1. Adds AVS kql queries for the first 4 items
2. minor grammar fixes
3. removes misplaced avs-1.kql file from the AVS root directory

### Breaking Changes

1. *Replace me*
2. *Replace me*

## As part of this Pull Request I have

- [ x] Read the [Contribution Guide](https://azure.github.io/Azure-Proactive-Resiliency-Library/contributing) and ensured this PR is compliant with the guide
- [ x] Checked for duplicate [Pull Requests](https://github.com/Azure/Azure-Proactive-Resiliency-Library/pulls)
- [ x] Associated it with relevant [GitHub Issues](https://github.com/Azure/Azure-Proactive-Resiliency-Library/issues) or ADO Work Items (Internal Only)
- [x ] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/Azure-Proactive-Resiliency-Library/tree/main)
- [ ] Ensured PR tests are passing
- [x ] Performed testing and provided evidence (e.g. screenshot of output) for any changes associated to ARG queries and/or scripts
- [x ] Updated relevant and associated documentation (e.g. Contribution Guide, Docs etc.)

### CODE REVIEW

**Comment by sblair01** [comment]
Looks good to me!

**Comment by michielvanschaik** [comment]
Looks good to me too!

**Review by ejhenry — COMMENTED** [review]
Thanks for the contribution @jchancellor-ms. For the ARG queries please include screenshots with the results of your testing. Thanks.

**Comment by jchancellor-ms** [comment]
avs-3 - 
image

**Comment by jchancellor-ms** [comment]
avs-4 - 
image

**Comment by jchancellor-ms** [comment]
avs-5 - 
image

**Comment by jchancellor-ms** [comment]
avs-6 - 
image

**Comment by jchancellor-ms** [comment]
> Thanks for the contribution @jchancellor-ms. For the ARG queries please include screenshots with the results of your testing. Thanks.

@ejhenry - added 1 screenshot per query as comments.  Please let me know if that is sufficient for review.

File: docs/content/services/specialized-workloads/azure-vmware-solution/_index.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ The presented resiliency recommendations in this guidance include Azure VMware S
 |[AVS-4 Enable Stretched Clusters for High Availability of the vSAN Datastore](#avs-4---enable-stretched-clusters-for-high-availability-of-the-vsan-datastore) | Availability | Low | Preview | No |
 |[AVS-5 Monitor CPU Utilization to ensure sufficient resources](#avs-5---monitor-cpu-utilization-to-ensure-sufficient-resources) | Monitoring | Medium | Preview | No |
```
Comment (ejhenry) [inline docs/content/services/specialized-workloads/azure-vmware-solution/_index.md:19]:
```suggestion
|[AVS-5 Monitor CPU Utilization to ensure sufficient resources](#avs-5---monitor-cpu-utilization-to-ensure-sufficient-resources) | Monitoring | Medium | Preview | Yes |
```

File: docs/content/services/specialized-workloads/azure-vmware-solution/_index.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ The presented resiliency recommendations in this guidance include Azure VMware S
 |[AVS-4 Enable Stretched Clusters for High Availability of the vSAN Datastore](#avs-4---enable-stretched-clusters-for-high-availability-of-the-vsan-datastore) | Availability | Low | Preview | No |
```
Comment (ejhenry) [inline docs/content/services/specialized-workloads/azure-vmware-solution/_index.md:18]:
```suggestion
|[AVS-4 Enable Stretched Clusters for High Availability of the vSAN Datastore](#avs-4---enable-stretched-clusters-for-high-availability-of-the-vsan-datastore) | Availability | Low | Preview | Yes |
```

File: docs/content/services/specialized-workloads/azure-vmware-solution/_index.md [hunk 3]
```diff
@@ -15,10 +15,10 @@ The presented resiliency recommendations in this guidance include Azure VMware S
 |  Recommendation                                   |      Category         |  Impact         |  State            | ARG Query Available |
 | :------------------------------------------------ | :---------------------------------------------------------------------: | :------:        | :------:          | :------:          |
 |[AVS-3 Configure Azure Monitor Alert warning thresholds for vSAN datastore consumption](#avs-3---configure-azure-monitor-alert-warning-thresholds-for-vsan-datastore-consumption) | Monitoring | High | Preview | No |
```
Comment (ejhenry) [inline docs/content/services/specialized-workloads/azure-vmware-solution/_index.md:17]:
```suggestion
|[AVS-3 Configure Azure Monitor Alert warning thresholds for vSAN datastore consumption](#avs-3---configure-azure-monitor-alert-warning-thresholds-for-vsan-datastore-consumption) | Monitoring | High | Preview | Yes |
```

**Review by ejhenry — APPROVED** [review]
LGTM 👍

**Comment by jchancellor-ms** [comment]
@ejhenry - do you need me to add the table changes you flagged to the commit?

**Comment by ejhenry** [comment]
> @ejhenry - do you need me to add the table changes you flagged to the commit?

@jchancellor-ms - I already committed the changes, so all good there. We just need the approval from a second APRL admin to merge.

**Review by oZakari — APPROVED** [review]

**Pull request merged**